.circleci/config.yml

```diff
@@ -78,6 +85,21 @@ commands:
       - run: SIMDJSON_FORCE_IMPLEMENTATION=fallback ctest $CTEST_FLAGS -L per_implementation
       - run: ctest $CTEST_FLAGS -LE "acceptance|per_implementation" # Everything we haven't run yet, run now.
 
+
+  cmake_test_all_noprep:
```

Comment:
I presume this is because that image already has cmake?

Don't worry about changing this in this PR unless you have some other things to change and decide you are inclined to slide it in, but here's my thought: let's change cmake_prep to install_cmake and add it to the steps for all the existing runs.

Sharing code in these things is kinda hard.

Reply:
Fixed in follow-up commit.